src/app/core/signals/remote_signals/community.nim

```diff
@@ -63,6 +64,15 @@ type DiscordChannelImportFinishedSignal* = ref object of Signal
   communityId*: string
   channelId*: string
 
+type CommunityTokenTransactionStatusChangedSignal* = ref object of Signal
+  transactionType*: string
+  success*: bool
+  hash*: string
+  communityToken*: CommunityTokenDto
```

Comment:
wondering why we need 3 token fields. 
Just to understand, do we ever mint/airdrop more than a single token per transaction? I know we mint both the Owner and TM tokens in a single "user action", but those are two separate transactions and two separate events coming from status-go, right?

Reply:
hmm after looking around for a bit, I might be wrong on that one, seems the 2 special tokens belong to the same contract

Reply:
Nope, owner token and master token deployment are done in a single transaction. We have got a deployer contract to do this. In case of owner token & master token deployment 2 fields are filled in a signal.
If this is community token deployment (or any operation made by a token, eg. burn, mint), communityToken field is filled.